app/views/org_admin/shared/_theme_selector.html.erb

```diff
@@ -0,0 +1,28 @@
+<%# locals: all_themes %>
+<div class="form-group">
+  <%= f.label _('Themes'), for: :theme_ids, class: 'control-label' %>
+  <div class="row">
+    <% if all_themes.length > 0 %>
+      <% 
+        cntr = 0
+        nbr_of_cols = (all_themes.length.to_f / MAX_NUMBER_THEMES_PER_COLUMN.to_f).ceil
+        col_size = (12 / (nbr_of_cols > 4 ? 3 : nbr_of_cols)).round
```

Comment:
This could be problematic for instances with a number of themes less than the MAX_NUMBER_THEMES_PER_COLUMN. That would mean the size of the column would be 1 and therefore the theme label would appear cut.

Reply:
Using the modulo here the syntax is simplified and cover every case. This would be my proposal:

```
<div class="form-group">
  <%= f.label _('Themes'), for: :theme_ids, class: 'control-label' %>
  <div class="row">
    <% if all_themes.length > 0 %>
      <% group_nbr = -1 %>
      <% all_themes.each_with_index do |theme, i| %>
        <% if i % MAX_NUMBER_THEMES_PER_COLUMN == 0 %>
          <div class="col-md-4">
          <% group_nbr += 1 %>
        <% end %>
        <div class="checkbox" data-toggle="tooltip" data-html="true" title="<%= theme.description %>">
          <%= f.label :theme_ids, raw("#{f.check_box :theme_ids, {multiple: true}, theme.id, nil} #{theme.title}") %>
        </div>
        <% if (i - group_nbr * MAX_NUMBER_THEMES_PER_COLUMN) == (MAX_NUMBER_THEMES_PER_COLUMN - 1) %>
          </div>
        <% end %>
      <% end %>
    <% else %>
      <p><%= _('No themes have been defined. Please contact your administrator for assistance.') %></p>
    <% end %>
  </div>
</div>
```

 do you mind to try it?

Reply:
sure. will give it a try.

Reply:
tested both with different array lengths and they both produce the same results. The label does not get clipped when there is only one column. Going to leave as is.